geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_config-schema.yaml

```diff
@@ -11,6 +11,17 @@ mapping:
     required: True
     sequence:
       - type: str
+  interpreted:
+    type: map
+    required: True
+    mapping:
+      regex;.*:
```

Comment:
`:` instead of `;` ?

Reply:
no it's effectively a `;` as the other regex :-)